js/lib/comments.js

```diff
@@ -113,6 +113,10 @@ elgg.comments.init = function() {
 		dc.toggleEdit();
 		return false;
 	});
+
+	// Highlight a comment if its id is found from the URL
+	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
```

Comment:
Huh, I don't think I had ever heard of this API... always just used location.href